src/Our.Umbraco.DocTypeGridEditor/Web/Controllers/DocTypeGridEditorApiController.cs

```diff
@@ -85,5 +91,37 @@ public object GetDataTypePreValues(string dtdId)
             var propEditor = PropertyEditorResolver.Current.GetByAlias(dtd.PropertyEditorAlias);
             return propEditor.PreValueEditor.ConvertDbToEditor(propEditor.DefaultPreValues, preValue);
         }
+
+        [HttpPost]
+        public HttpResponseMessage GetPreviewMarkup([FromBody] FormDataCollection item, [FromUri] int nodeId)
+        {
+            // Get page container node
+            var page = UmbracoContext.ContentCache.GetById(nodeId);
+            if (page == null)
+            {
+                // If unpublished, then fake PublishedContent (with IContent object)
+                page = new UnpublishedContent(nodeId, Services);
+            }
+
+            var culture = UmbracoContext.Application.Services.ContentService.GetById(nodeId).GetCulture();
```

Comment:
I'm curious about this call. Wondering if there's a lighter way to get the culture for a specific node?